### ISSUE TITLE
Tenacity (64-bit Windows, build 657a83) fails to load modern FFmpeg libraries

### ISSUE THREAD

**Issue by ghost** [tenacityteam/tenacity-legacy#425]
- [x] I have read the specified guidelines for issues

**Describe the bug**
Tenacity is unable to load and use the latest FFmpeg shared library (currently version 4.4).

**To Reproduce**
1. Download the Tenacity build artifact for Windows 64-bit (https://github.com/tenacityteam/tenacity/suites/3376600916/artifacts/79563150)
2. Download the FFmpeg 4.4 shared library from SourceForge (https://sourceforge.net/projects/ffmpeg-windows-builds/files/release/n4.4/ffmpeg-N-101785-gdc91b913b6-win64-shared.7z/download) and extract it to `C:\Program Files\FFmpeg for Tenacity`
- Note: Tenacity seems to hardcode which versions of the FFmpeg DLLs it desires. For example, you will need to rename `avcodec-58.dll` to `avcodec-55.dll` for Tenacity to accept it. 
3. Open Tenacity
4. Go to Edit > Preferences
5. Click "Libraries"
6. Click "Locate...", then click "Browse..." and navigate to the `C:\Program Files\FFmpeg for Tenacity` directory
7. Click Open, and then click OK
8. Tenacity returns to Preferences: Libraries and reports "FFmpeg library not found"

**Expected behavior**
Tenacity should return to the Preferences: Libraries and report the FFmpeg library version, as well as actually load it.

**Screenshots**

Before loading the FFmpeg library: ![image](https://user-images.githubusercontent.com/9056756/127788322-e65b6463-b248-4152-9565-f6a3633192b9.png)

Popup window showing the file path Tenacity uses to find the FFmpeg library: ![image](https://user-images.githubusercontent.com/9056756/127788329-e603624c-8120-402c-8420-308d9588ca9b.png)

Explorer window for manually specifying the FFmpeg library path: ![image](https://user-images.githubusercontent.com/9056756/127788493-c4fccd4a-4366-4634-8070-454e72015c69.png)

After attempting to load the FFmpeg library (same image used, as nothing was loaded): ![image](https://user-images.githubusercontent.com/9056756/127788322-e65b6463-b248-4152-9565-f6a3633192b9.png)

**Additional information (please complete the following information):**
 - OS: Windows 11 build 22000.100
 - Version: Tenacity 657a83, which is approximately 3.0.3 or 3.0.4?

**Additional context**

- Tenacity should not hardcode which versions of the FFmpeg libraries it requires
- Tenacity does not provide a visible error message when the FFmpeg library fails to load. Errors are only present in `%appdata%\tenacity\lastlog.txt` (which may require closing Tenacity if the file does not yet exist)
- This issue has not been tested with Tenacity 32-bit using 32-bit FFmpeg libraries
- Here is the log file: [lastlog.txt](https://github.com/tenacityteam/tenacity/files/6913621/lastlog.txt)

**Comment by n0toose** [comment]
Not of huge relevance, if one keeps the problems laid out in the "Additional context" section in mind, but a user reported to me that this did work for them on Windows 10: https://lame.buanzo.org/ffmpeg64audacity.php

**Comment by rubyFeedback** [comment]
> Not of huge relevance

I don't disagree, but small UI improvements may be useful in the long run, in particular the part about
error notifications when something could not be found.

In ruby-gtk3 I use the popover widget functionality for quick error reporting, which is quite convenient.
Not sure how the native windows UI work but I suppose there may be some quick and simple notification
too. Or, even simpler, the string could be extended and/or perhaps in a slightly different colour or something
like that. (I am not a UI guru so I try to keep everything simple at all times in my own code bases.)

**Comment by JPcode05** [comment]
@citrusui
1. The program's UI looks blurred (on a high-res display) #168 
2. It looks like you're using Windows 11!
edit: Sorry this doesn't answer the FFmpeg question.

**Comment by n0toose** [comment]
> > Not of huge relevance
> 
> I don't disagree, but small UI improvements may be useful in the long run...

I agree with you wholeheartedly, but I was talking about how another person dealt with a similar problem such as the one that @citrusui is dealing with. Was wondering if that could help for now. We'll keep what you just wrote in the back of our heads too.

**Comment by Be-ing** [comment]
Is this still relevant since #228 was merged? Tenacity is now built with an up to date FFmpeg library.

**Comment by AnotherFoxGuy** [comment]
Last time I checked it was still broken

**Comment by kubo6472** [comment]
Working for me on Win10 21H1, using [this build](https://nightly.link/tenacityteam/tenacity/workflows/cmake_build/master/Tenacity_Windows_64bit_1196324002_.zip) and [this ffmpeg](https://phoenixnap.dl.sourceforge.net/project/ffmpeg-windows-builds/release/n4.4/ffmpeg-N-101785-gdc91b913b6-win64-shared.7z)
